webapp-django/crashstats/settings/base.py

```diff
@@ -643,11 +644,22 @@ def COMPRESS_JINJA2_GET_ENVIRONMENT():
 # then set this to False
 SESSION_COOKIE_SECURE = config('SESSION_COOKIE_SECURE', True, cast=bool)
 
-# To get your Sentry key, go to https://errormill.mozilla.org/
-RAVEN_CONFIG = {
-    'dsn': config('RAVEN_DSN', '')  # see https://errormill.mozilla.org/
-}
-
+# When socorro is installed (python setup.py install), it will create
+# a file in site-packages for socorro called "socorro/socorro_revision.txt".
+# If this socorro was installed like that, let's pick it up and use it.
+try:
+    SOCORRO_REVISION = resource_string('socorro', 'socorro_revision.txt')
+except IOError:
+    SOCORRO_REVISION = None
+
+# Raven sends errors to Sentry.
+# The release is optional.
+raven_dsn = config('RAVEN_DSN', '')
+if raven_dsn:
+    RAVEN_CONFIG = {
+        'dsn': raven_dsn,
+        'release': SOCORRO_REVISION,
+    }
```

Comment:
Note, if you don't define the `RAVEN_CONFIG` variable here, the `raven.contrib.django.raven_compat` app is smart enough to back off and not set up raven.